### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `typos --format brief`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ $ mdcat sample.md
 
 ![mdcat showcase with different colour themes][sxs]
 
-mdcat in [WezTerm], with "One Light (base16)", "Gruvbox Light", and "Darcula
+mdcat in [WezTerm], with "One Light (base16)", "Gruvbox Light", and "Dracula
```
Comment (swsnr) [inline README.md:11]: This was intentional, see https://wezfurlong.org/wezterm/colorschemes/d/index.html?h=darcula#darcula-base16

**Review by swsnr — CHANGES_REQUESTED** [review]
Thanks a lot.  Pretty embarrassing too 😅

However the name of the colour theme was nommistake, in my defense.  Please do restore the old spelling here 🙂

**Comment by szepeviktor** [comment]
There is one more typo.
https://github.com/swsnr/mdcat/blob/64b01a6aae1e1dcce356ae880e0010dbde84e025/.github/workflows/test.yml#L51
_implicity_

**Comment by kianmeng** [comment]
@swsnr Do you want to add the configuration file for `typos` as suggested by @szepeviktor?
See https://github.com/kianmeng/mdcat/commit/08d66202627cc923365730e174fc0c39336de5b7#commitcomment-130857103

**Comment by swsnr** [comment]
I don't know what `typos` is.  If it's some kind of automated spell checker, I don't think so.  I don't think catching the occasional typo in a comment is worth the effort to have yet any another tool.

**Review by swsnr — APPROVED** [review]

**Comment by szepeviktor** [comment]
> worth the effort

@swsnr I am interested in your decision process.
Are you low on resources for operations? Is it behind your decision?

**Comment by swsnr** [comment]
There's no "process", I just don't particularly care about typos.  It's not as if a few typos cause any troubles…

**Comment by szepeviktor** [comment]
Here you go: viktor-devops for you ✈️ 
https://github.com/szepeviktor/typos-on-mdcat/blob/master/.github/workflows/spelling.yml#L7

**Comment by swsnr** [comment]
@szepeviktor As said, I do not really care about typos, so please don't open pull requests for every typo you find.  That's just wasting my time, and is only going to annoy me.

**Comment by szepeviktor** [comment]
All right. I will postpone typo PR-s with 1 year.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**